server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -139,22 +139,22 @@ const { defaultExpensesDAO, jurorBankDetailsDAO } = require('../../../objects/ex
 
   module.exports.getOverviewTab = function(app) {
     return function(req, res) {
-      var successCB = function(response) {
+      var successCB = async function(response) {
```

Comment:
Because this is a `Promise.all` callback now, can we make this `function([overview, detail])` or equivalent please, rather than reference into an array throughout the function.

Reply:
updated